### ISSUE TITLE
Cannot use noise data if flagged noise_is_continuous=False

### ISSUE THREAD

**Issue by joefowler** [usnistgov/mass#157]
**[Original report](https://bitbucket.org/joe_fowler/mass/issues/157) by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

```python
data = mass.TESGroup("file.ljh", "noise.noi", noise_is_continuous=False)
data.compute_noise()
```

will fail with an error having an ultimate source like this:

```text
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-233-e4b07d10cb6f> in <module>()
----> 1 ds.noise_records.compute_power_spectrum_reshape(max_excursion=1000, seg_length=ds.nSamples)

/usr/local/lib/python2.7/dist-packages/mass-0.6.4-py2.7-linux-x86_64.egg/mass/core/channel.pyc in compute_power_spectrum_reshape(self, window, seg_length, max_excursion)
    164 
    165         if not self.continuous and seg_length is not None:
--> 166             raise ValueError("This NoiseRecords doesn't have continuous noise; it can't be resegmented.")
    167 
    168         if seg_length is None:

ValueError: This NoiseRecords doesn't have continuous noise; it can't be resegmented.
> /usr/local/lib/python2.7/dist-packages/mass-0.6.4-py2.7-linux-x86_64.egg/mass/core/channel.py(166)compute_power_spectrum_reshape()
    165         if not self.continuous and seg_length is not None:
--> 166             raise ValueError("This NoiseRecords doesn't have continuous noise; it can't be resegmented.")
    167 

```

I think the fix is simple, though I'm not sure a test case is.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Fix bugs when using non-continuous noise records

Fixes #157, with test.

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Merged in fowler (pull request #62)

Fix #157: non-continuous noise files

Approved-by: Joseph Fowler <drjoefowler@gmail.com>

**Comment by joefowler** [comment]
**Original comment by Joseph Fowler (Bitbucket: [joe_fowler](https://bitbucket.org/joe_fowler), ).**

----------------------------------------

Removing milestone: v0.7 (automated comment)
